WorkloadManagementSystem/Agent/StatesMonitoringAgent.py

```diff
@@ -53,8 +53,9 @@ def initialize(self):
 
     self.reportPeriod = 120
     self.am_setOption("PollingTime", self.reportPeriod)
+    self.messagequeue = self.am_getOption('MessageQueue', 'dirac.wmshistory')
```

Comment:
You have to update the ConfigTemplate.cfg of WorkloadManagementSystem, as this is what makes the test to fail. 
At the same time you should use the `literalinclude`magic to create the docstring for this module.